src/client/Microsoft.Identity.Client/Platforms/netcore/NetCorePlatformProxy.cs

```diff
@@ -201,6 +201,11 @@ public override Task StartDefaultOsBrowserAsync(string url)
                                 RedirectStandardOutput = true,
                                 RedirectStandardError = true
                             };
+                            string sudoUser = Environment.GetEnvironmentVariable("SUDO_USER");
```

Comment:
1. Is this a standard way to find if user is sudo?
2. I think this block can be moved to outside the foreach loop?

Reply:
Yes, this is a well known convention on Linux. Agree on moving it outside the block.

Reply:
Done!